int32 cust_x
string cust_string
